inference-engine/tests/functional/plugin/gna/pass_tests/convert_padded_to_valid_conv.cpp

```diff
@@ -277,8 +277,8 @@ const std::vector<std::vector<size_t >> maxpool1DPools = {{1, 2}};
 const std::vector<std::vector<size_t >> maxpool1DStrides = {{1, 1}};
 
 const std::vector<std::vector<size_t>> input2DNHWC = {{1, 16, 16, 32}};
-const std::vector<std::vector<size_t >> kernels2D = {{2, 2}, {4, 1}, {1, 3}};
-const std::vector<std::vector<size_t >> strides2D = {{1, 1}, {1, 2}, {2, 1}, {2, 2}};
+const std::vector<std::vector<size_t >> kernels2D = {{2, 2}, {4, 1}};
+const std::vector<std::vector<size_t >> strides2D = {{1, 1}, {2, 1}};
```

Comment:
Too many tests removed, I understand some combinations are invalid,
but what about:
kernel 2,2 stride 1,2
kernel 2,2 stride 2,2
kernel 1,3 stride 1,1
kernel 1,3 stride 1,2
?

Reply:
> Too many tests removed, I understand some combinations are invalid,
> but what about:
> kernel 2,2 stride 1,2
> kernel 2,2 stride 2,2
> kernel 1,3 stride 1,1
> kernel 1,3 stride 1,2
> ?

You're right that these combinations are valid, but I would have to create a new test case to include them and this does not really add much value to padding tests, because the padding isn't calculated by the tested code, but by ngraph and stride/kernel combinations don't influence the transformation behavior. It matters more for 2d->1d tests.